### DISCUSSION TITLE
Commissioned Art for Tuxemon

### DISCUSSION THREAD

**Discussion by Sanglorian** [Tuxemon/Tuxemon#2325] [General]
Hi folks, have just added a 3D model of Agnite that I commissioned on Fiverr [to the Printables website](https://www.printables.com/model/949792-tuxemon-cute-fiery-lizard-agnite). It joins [Mystikapi, Turnipper and Nudiflot](https://www.printables.com/@Sanglorian_281874/models). 

![render mode](https://github.com/user-attachments/assets/393e6df9-49c5-48d3-bf4e-70c1335af2a2)

**Comment by Sanglorian** [comment]
A couple more pieces, these from [Nikai Region](https://www.deviantart.com/nikai-region/). 

![Commission - Spyder Rockie - Artwork](https://github.com/user-attachments/assets/51057857-b99a-4498-9590-c4519b0ecb81)

![Commission - Foofle - Artwork](https://github.com/user-attachments/assets/152a649a-091f-40f8-a28b-851e722985b3)

**Reply by Sanglorian** [reply]
Three more pieces from Nikai Region:

![Commission - Gemblin - Artwork resize](https://github.com/user-attachments/assets/e84bd4b7-c01c-4015-a149-4fed6edb2c19)

![Commission - Scoop Lady - Artwork](https://github.com/user-attachments/assets/e13f1b22-5c5c-46c7-8219-b939d9b1dfed)

![Commission - Treasurblin - Artwork](https://github.com/user-attachments/assets/24afcfaa-0e24-4cec-95db-169cc958e9fe)

**Reply by JaskRendix** [reply]
@Sanglorian the monsters from the Nikai Region have a name? If not what do you think about these?

Name: Dustpuff
Name Origin: Derived from "dust" and "puff," suggesting its ability to create clouds of dust.

Name: Spikelet
Name Origin: Derived from "spike," referring to its sharp protrusions, and "let," a diminutive suffix suggesting small size.

Name: Chesttaur
Name Origin: A combination of "chest," referencing its body shape resembling a treasure chest, and "taurus," the Latin word for bull, alluding to its strength and imposing stature.

**Reply by Sanglorian** [reply]
Thanks @JaskRendix but those are actually reimaginings of existing monsters! Foofle, Gemblin and Treasurblin.

**Comment by Sanglorian** [comment]
Commissioned a 3D model of Axylightl. 
![axylightl](https://github.com/user-attachments/assets/a51c10fe-64bb-45c2-8baa-3dd8b974feda)


https://www.printables.com/model/996272-tuxemon-cute-light-axolotl-axylightl

**Comment by Sanglorian** [comment]
A couple more artworks from Nikai Region that I purchased.

These are original, so welcome any thoughts on what we might call them, what blurbs we would give them, etc. 

![Lab Rat - First Stage - Artwork](https://github.com/user-attachments/assets/2a6d2276-e39c-4a88-953c-645e57856d42)

![Lab Rat - Zombie Rat - Evolution - Artwork](https://github.com/user-attachments/assets/3368e51f-3a31-4e6b-a526-e566edfb4a6d)

**Reply by HippasusTwo** [reply]
I like the skull dude, it reminds me of a critter I had daydreamed of once that was very similar except it had a scythe for a tail.

It could be a reference to horned gophers, although the first critter has a mousy tail the large nose does remind me of a gopher. Gophermouse could be a name for it? Or, huh, Gophmouse almost sounds like ghost mouse.
Then the evo could be Gophst
Horned gophers are extinct [rodents](https://en.wikipedia.org/wiki/Rodent) from the genus Ceratogaulus, a member of the extinct [fossorial](https://en.wikipedia.org/wiki/Fossorial) rodent clade [Mylagaulidae](https://en.wikipedia.org/wiki/Mylagaulidae). Ceratogaulus is the only known rodent genus with [horns](https://en.wikipedia.org/wiki/Horn_(anatomy)), and is the smallest known horned mammal.

**Reply by Sanglorian** [reply]
Lovely, thanks! Have added to the wiki under those names.

**Comment by Sanglorian** [comment]
I commissioned a 3D sculpt of Memnomnom, and tried my hand at sculpting Poinchin and Katapill. 
On Printables: https://www.printables.com/model/1255973-tuxemon-memnomnom-ghost-cat-poinchin-sea-urchin-ka/files

**Comment by Sanglorian** [comment]
My friend printed and painted a bunch of the Tuxemon STLs I had already commissioned -- I've added photos to the STLs [I'd uploaded to Printables](https://www.printables.com/@Sanglorian_281874).

I also used the Maker World AI tools to create eight more, which he printed and painted too -- they're [up now on the Printables site](https://www.printables.com/model/1352910-tuxemon-set-cute-creatures-for-a-monster-catching).

**Comment by Sanglorian** [comment]
I commissioned an idle animation from Ernest Rivera -- I'm really happy with it:

![Final Animation-256](https://github.com/user-attachments/assets/5b43c3e8-76b3-4ded-beb1-0e3cce5ba929)

Would it work as the opening screen of the game, replacing the current black? When the player hit start, it could go to the menu (save, load, options, etc.) Perhaps the Python and CC BY-SA logos could appear on the menu screen rather than the opening screen? 

For music, I imagine something like [http://opengameart.org/content/the-adventure-begins-8-bit-remix The Adventure Begins] or maybe [https://opengameart.org/content/jrpg-collection the JRPG theme from this collection]?

**Reply by HippasusTwo** [reply]
Ooh, that's really cool. Yeah, that seems like a nice opening. Someone else was working on end credits if I recall correctly.

Soothing style music would seem to suit it, so the JRPG_maintheme file would seem to suit it.

**Reply by ultidonki** [reply]
Amazing! It's a Dandicub, an Aardorn, a Katapill and a Cardi(wing?)!!! It looks great! :smile:
